src/SprykerSdk/Zed/Integrator/Business/ManifestStrategy/CopyModuleFileManifestStrategy.php

```diff
@@ -89,6 +90,6 @@ protected function getTargetPath(array $manifest): string
      */
     protected function camelCaseToDash(string $value): string
     {
-        return (new CamelCaseToSeparator('-'))->filter($value);
+        return (new UtilTextService())->camelCaseToDash($value);
```

Comment:
Binding the tool too much to modules of core will create a chicken-egg problem IMO.